ramanujantools/cmf/cmf_test.py

```diff
@@ -155,3 +155,11 @@ def test_substitute_trajectory_walk_equivalence():
     assert subbed.walk({n: 1}, iterations, {n: 1}) == unsubbed.walk(
         trajectory, iterations, start
     )
+
```

Comment:
We definitely want a test that calculates delta of a CMF and compares it to an outside implementation of delta (the normal trajectory matrix -> limit -> delta flow)